DATABASE.md

```diff
@@ -0,0 +1,22 @@
+### Install
+
+Install [postgres server version 9.6](https://github.com/PostgresApp/PostgresApp/releases/download/v2.2/Postgres-2.2-9.5-9.6-10-11.dmg)
+
+This version defaults to starting postgres server version 11. To change to version 9.6:
```

Comment:
Why version 9.6?

Reply:
that is the latest supported by google could sql